src/netaddress.cpp

```diff
@@ -101,24 +194,82 @@ bool CNetAddr::SetInternal(const std::string &name)
     return true;
 }
 
+namespace torv3 {
```

Comment:
I see namespace is - to my understanding - very seldom used in the project, is there a specific reason it has to be used here?

Reply:
The primary reason to use a namespace - things belong together.

Every time I spot myself prefixing symbols with a common prefix I ask myself whether I am implementing a poor man's namespace. In this case, without a namespace, I would have had to prefix all the symbols with a `torv3_` prefix. E.g. `TORV3_CHECKSUM_LEN`, `TORV3_VERSION`, `Torv3Checksum()`.

Reply:
I completely agree, I think it makes things more contained, well structured and easier to comprehend. It seems to me a lot of new code would benefit from namespacing and perhaps worth considering when touching old code as well?